src/inclusion.js

```diff
@@ -32,12 +35,15 @@ export const link = (proof) =>
  *
  * @param {API.ByteView<API.InclusionProof, typeof CBOR.code>} bytes
  */
-export const decode = (bytes) => {
-  const [tree, index] = CBOR.decode(bytes)
+export const decode = (bytes) => from(CBOR.decode(bytes))
 
-  // Note we need to go through this to ensure the bigint conversion
-  return create({ tree: Proof.from(tree), index: Proof.from(index) })
-}
+/**
+ * Takes proof in somewhat arbitrary form and returns a proof data.
```

Comment:
is the `somewhat arbitrary form` here a dig at the filecoin encoded form? Consider dropping that phrase, or expanding the explanation. without more context it's confusing.

```suggestion
 * Takes proof in either object or array form and returns a proof data.
```